ofl/notokufiarabic/METADATA.pb

```diff
@@ -0,0 +1,20 @@
+name: "Noto Kufi Arabic"
+designer: "UNKNOWN"
```

Comment:
Let's use `designer: "Google"` until we compile data on who the actual designers were

Reply:
used Google LLC